_docs/_latest/configure/scanner/default_rules.md

```diff
@@ -1,6 +1,6 @@
 ---
 title: Default Rules
-order: 106
+order: 302
```

Comment:
nit: Can we be consistent use the "-" for spaces in filenames instead of "_"

Reply:
Filename updated.